Makefile

```
@@ -35,6 +35,9 @@ help: ## Show this help message
 
 init: | $(BIN_DIR)/grype $(BIN_DIR)/syft node_modules/ ## Initialize the project dependencies
 
+license-check: $(SBOM_FILE)
```

This Make command should be documented:

```suggestion
license-check: $(SBOM_FILE) ## Check dependency licenses
```